lib/api/core/notifier.js

```diff
@@ -340,11 +340,15 @@ class NotifierController {
           result._source,
           result._id);
 
+        
+        let updatedField = Object.keys(request.input.body);
+
         return this.notifyDocument(
           matchedRooms, request, 'in', 'done', 'update', {
             _meta: updatedDocument._meta || {},
             _id: updatedDocument._id,
-            _source: updatedDocument._source
+            _source: updatedDocument._source,
+            _updatedField: updatedField.filter(updatedField => !['_kuzzle_info'].includes(updatedField))
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') "updatedField" hides or potentially hides a variable declared in an outer scope at line 344. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AVariableShadowing)